### DISCUSSION TITLE
[NgRx 速成班] 簡介 NgRx | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#56] [Announcements]
# [NgRx 速成班] 簡介 NgRx | 全端開發人員天梯

NgRx 是一個強大，且功能完整的 Angular 狀態管理套件，在我們的應用程式越來越複雜的時候，它很適合用來管理一些全域的狀態，並透過適度的抽象化以及加入一定的規範，降低整體程式的耦合性，打造出更好維護及管理的程式碼。接下來的一系列文章將會介紹 NgRx 的核心精神，使用方式及相關工具。

https://fullstackladder.dev/blog/2022/04/17/ngrx-01-introduce/

**Comment by indianazhao** [comment]
很棒的介紹文！期待下一篇！

**Reply by wellwind** [reply]
感謝肯定 ^^
寫完這篇後剛好 Angular 14 即將出現，結果精力都花在新東西上
最近又剛好比較忙，假日都在唸書，等我忙得差不多會盡快把文章補齊的 @@

**Comment by bp4m4h94** [comment]
感謝技術解析!!!
順帶一提，StudentStore 範例程式碼似乎有錯字，getStore應改為getScore。
